autoincfield.php

```diff
@@ -52,13 +50,14 @@ function autoincfield_civicrm_buildForm($formName, &$form) {
   if ($formName == 'CRM_Custom_Form_Field') {
     if ($form->elementExists('data_type')) {
       // Add autoincfield js
-      CRM_Core_Resources::singleton()->addScriptFile('com.joineryhq.autoincfield', 'js/autoincfield.js', 100, 'page-footer');
+      CRM_Core_Resources::singleton()->addScriptFile('com.joineryhq.autoincfield', 'js/CRM_Custom_Form_Field.js', 100, 'page-footer');
 
       // Create necessary fields
-      $form->addElement('checkbox', 'autoinc', ts('Is Autoincrement?'));
-      $form->addElement('text', 'min_value', ts('Minimum next value'));
+      $form->addElement('checkbox', 'autoinc', E::ts('Is Autoincrement?'));
+      $form->addElement('text', 'min_value', E::ts('Minimum next value'));
       // Assign bhfe fields to the template.
       $tpl = CRM_Core_Smarty::singleton();
+      $tpl = CRM_Core_Smarty::singleton();
```

Comment:
@lucky091588 Looks like this line is redundant. Can you remove it from the PR? Thanks.

Reply:
Done